CMakeLists.txt

```diff
@@ -4,9 +4,8 @@
 cmake_minimum_required(VERSION 2.8)
 project(zeq)
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
-  ${PROJECT_SOURCE_DIR}/CMake/common)
+  ${CMAKE_SOURCE_DIR}/CMake/common)
```

Comment:
Yes, this is correct. The alternative currently in place is almost unusable when you need to change something in CMake/common, since you have to copy the file everywhere.